### PR TITLE
docs: add ECR docs

### LINKED ISSUE #105: ECR support
Hello,

I tried using ORAS with ECR support and I get `Error: unexpected response: 404 Not Found`
I know the project is still in its infancy but I wanted to hack on it in AWS and I was wondering if anyone got it to work. Anything I can do to help test this? :)

### PR DESCRIPTION
**Pull request by RichiCoder1**
Adds a reference to ECR support and the blog post.

Closes #105;

### CODE REVIEW

File: implementors.md [hunk 1]
```diff
@@ -173,6 +174,30 @@ Then you can pull files from the registry or push files to the registry.
...
   oras logout <registry-ip>:5000
   ```
+  
+### [Amazon Elastic Container Registry (ECR)](https://aws.amazon.com/ecr/)
+
+ECR Artifact Blog Post: <https://aws.amazon.com/blogs/containers/oci-artifact-support-in-amazon-ecr/>
+
+- Authenticating with ECR using the AWS CLI
+  ```sh
+  oras login $REPO_URI --username AWS --pasword $(aws ecr get-login-password)
```
Comment (zmarouf) [inline implementors.md:184]: There seems to be a typo.
```suggestion
  oras login $REPO_URI --username AWS --password $(aws ecr get-login-password)
```

Comment (zmarouf) [reply implementors.md:184]: May I also suggest
```suggestion
  aws ecr get-login-password --region $AWS_REGION --profile $PROFILE | oras login \
        --password-stdin \
        --username AWS \
        "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"
```

**Comment by zmarouf** [comment]
@RichiCoder1 Thanks for this and for pinging on #105, I was not aware of the recent change.
According to the AWS docs, authentication to a repo via AWS CLI v2 is done like so
```
aws ecr get-login-password --region $AWS_REGION --profile $PROFILE | oras login \
        --password-stdin \
        --username AWS \
        "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"
```
It prevents this ugly warning:

> WARNING! Using --password via the CLI is insecure. Use --password-stdin.

**Comment by RichiCoder1** [comment]
> @RichiCoder1 Thanks for this and for pinging on #105, I was not aware of the recent change.
> According to the AWS docs, authentication to a repo via AWS CLI v2 is done like so
> ```
> aws ecr get-login-password --region $AWS_REGION --profile $PROFILE | oras login \
>         --password-stdin \
>         --username AWS \
>         "${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"
> ````
> It prevents this ugly warning:
> 
> > WARNING! Using --password via the CLI is insecure. Use --password-stdin.

I think I had originally leaned towards keeping the example concise, but it's probably much better with that full example.

**Comment by zmarouf** [comment]
Thanks for the update. Meanwhile I opened a support ticket because I keep hitting `500 Internal Server Error`
So I don't know how stable this is :)

**Comment by RichiCoder1** [comment]
> Thanks for the update. Meanwhile I opened a support ticket because I keep hitting 500 Internal Server Error
> So I don't know how stable this is :)

Oof :\. I admit, this being AWS I expected it to be pretty rock solid from the outset. Hopefully you can quickly get to a solution!

**Comment by jdolitsky** [comment]
LGTM, will probably want somebody from Amazon to review, cc @omieomye @mhausenblas @jtoberon

**Review by mhausenblas — APPROVED** [review]
Thanks and LGTM!

**Review by jdolitsky — APPROVED** [review]

**Comment by jdolitsky** [comment]
@RichiCoder1 can you rebase when you have a minute?

**Comment by RichiCoder1** [comment]
@jdolitsky Rebased and up to date

**Pull request merged**